### ISSUE TITLE
Claims folder details element should be read as a button with expanding/collapsing functionality

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12143]
The Claims folder Details element is read as a tab for assistive technology users. This element is a button with an expanding and collapsing feature and should not be read as a tab. (New: 08/27/2019)

This component comes directly from the U.S. Web Design System, it might be worth reaching out to the FE designer on that system to ask about why they coded it that way (I can put in touch)
